src/pages/createProfile/CreateProfile.tsx

```diff
@@ -0,0 +1,59 @@
+import { FunctionComponent } from "react";
+import { Form, Formik } from "formik";
+import { Redirect, Route, Switch, useRouteMatch } from "react-router-dom";
+import * as Yup from "yup";
+
+interface FormValues {
+  // form value definitions here (placeholder added to prevent linting error)
+  readonly example: string;
+}
+
+const CreateProfile: FunctionComponent = () => {
+  const { path } = useRouteMatch();
+
+  const initialValues: FormValues = {
+    // initial values here
+    example: "",
+  };
+
+  const ValidationSchema = Yup.object().shape({});
+
+  const handleSubmit = () => {
+    // logic to submit form data here
+  };
+
+  return (
+    <Formik
+      initialValues={ initialValues }
+      validationSchema={ ValidationSchema }
+      onSubmit={ handleSubmit }
+    >
+      { () => (
+        <Form>
+          <Switch>
```

Comment:
Do we want to upgrade react-router-dom to V6? if so, we should do it now while the routing in our project is still pretty light. If we do, this would become ['Routes' instead of 'Switch'](https://reactrouter.com/docs/en/v6/upgrading/v5#upgrade-all-switch-elements-to-routes).

Reply:
Sounds good to me. We can make that upgrade in a separate PR